.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Description
+This PR…
+
+
+## Screenshots
+[Add screenshots here]
+
+
+## Changes
+[Add a list of features added/changed, bugs fixed etc]
+
+* Adds…
+* Changes…
+
+
+## Notes to reviewer
+[Add instructions for testing]
+
+
+## Related issues
```

Comment:
Perhaps add this, and maybe put this at the top, or just as the first line of the description? I see getting the issue linked as basically the most important thing as it cuts out extra work through automation. :-D

Use Github keywords to link to a related issue, e.g. fixes #123 or closes #123